examples/src/chains/conversational_qa.ts

```diff
@@ -5,6 +5,10 @@ import { OpenAIEmbeddings } from "langchain/embeddings/openai";
 import { RecursiveCharacterTextSplitter } from "langchain/text_splitter";
 import { BufferMemory } from "langchain/memory";
 import * as fs from "fs";
+import { PromptTemplate } from "langchain/prompts";
+import { RunnableSequence } from "langchain/schema/runnable";
+import { Document } from "langchain/document";
+import { StringOutputParser } from "langchain/schema/output_parser";
 
 export const run = async () => {
```

Comment:
You can remove `const run()` wrapping this, it used to be necessary but no longer is.